### PR TITLE
fix: UI_REACT: Change toolbar menu separators to spaces

### PR DESCRIPTION
**Pull request by jpav**
Fixes: #1783 
Fixes: [ENTESB-12730](https://issues.redhat.com/browse/ENTESB-12730)

### CODE REVIEW

File: ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx [hunk 1]
```diff
@@ -58,223 +58,211 @@ export const AtlasmapContextToolbar: FunctionComponent<IAtlasmapContextToolbarPr
...
+          <Button
+            variant={'plain'}
+            aria-label="Show/hide free view"
+            onClick={onToggleShowFreeView}
+          >
+            <PficonDragdropIcon />
+          </Button>
+        </Tooltip>
+      </ToolbarItem>
+      {showFreeView && (
```
Comment (riccardo-forina) [inline ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx]: I missed when the zoom buttons have been moved to the context toolbar, but I think their place is the control bar. I mean, it's exactly how the Topology component is showcased [in the doc](https://www.patternfly.org/v4/documentation/react/components/topology).

Comment (jpav) [reply ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx]: I understand, but this was a direct request from the UX team.

Comment (riccardo-forina) [reply ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx]: Maybe they missed out that detail, wdyt?

Comment (jpav) [reply ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx]: I doubt it. If you look at the JIRA above, they have a screenshot that specifically points to how it was laid out before, and how they want it moved. We could pull them into the conversation if you like.

Comment (riccardo-forina) [reply ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapContextToolbar.tsx]: Yeah sure. I mean we didn’t have the zoom buttons before and they make sense inside the canvas as per documentation. The other buttons are ok in the top toolbar

**Comment by jpav** [comment]
I went ahead and merged since this issue is specifically about the toolbar dividers and not the location of the zoom buttons. I've asked the UX team to setup a call where we can discuss the location issue related to the original issue: [ENTESB-12730](https://issues.redhat.com/browse/ENTESB-12730)

**Pull request merged**